components/ticket-card.tsx

```diff
@@ -7,17 +7,17 @@ import type { UserTicket } from '@/lib/api/types'
 
 interface TicketCardProps {
   ticket: UserTicket
-  ticketPrice?: number
   index: number
   totalTickets: number
 }
 
 export default function TicketCard({
   ticket,
-  ticketPrice,
   index,
   totalTickets,
 }: TicketCardProps) {
+  // Use ticket_price from ticket if available
+  const ticketPrice = ticket.ticket_price
```

Comment:
You don't need to create a constant, you can just use `ticket.ticket_price` if you need the value of the price of the ticket